planning:
  footprint:
    type: 'tricky_circle'
    inflation_scale: 1.0

  min_frontier_size: 20
  planning_epsilon: 1.0
  goal_check_radius: 0.5
  
  skip_pose: 1000
  skip_pixel: 50
  
  # robot_frame_id: 'husky/base'
  robot_frame_id: "semantic_segmentation_world"
  interpolation:
    alpha: 1
    radius: 8.0
